core/src/main/scala/cats/data/Tuple2K.scala

```diff
@@ -189,6 +189,22 @@ sealed abstract private[data] class Tuple2KInstances8 {
       def F: Functor[F] = FF
       def G: Functor[G] = GG
     }
+  implicit def catsDataSemigroupalForTuple2K[F[_], G[_]](implicit
+    FF: Semigroupal[F],
+    GG: Semigroupal[G]
+  ): Semigroupal[λ[α => Tuple2K[F, G, α]]] =
```

Comment:
any reason for not using the shorthand `*` here? :)

Reply:
Oh I see, it's not used in any of the others either, that's odd, but should be fine :)

Reply:
Yeah I was just trying to follow the style of the others, unless you want me to change them?

Reply:
no, it's all good! thanks!

Reply:
Alright sounds good :)